DEVELOPMENT.md

```diff
@@ -87,18 +87,11 @@ kubectl create clusterrolebinding cluster-admin-binding \
   --user="${USER}"
 ```
 
```

Comment:
We added the deploy Knative build because its a required dependency that should be installed

Reply:
for sure I definitely still want to explain how to install it but in a previous commit i added this step to the overall install instructions:

![image](https://user-images.githubusercontent.com/432502/46687260-50cb0d80-cbaf-11e8-8bf7-65a852001366.png)

and teardown:

![image](https://user-images.githubusercontent.com/432502/46687271-5aed0c00-cbaf-11e8-8e36-3e6c02bb4b9e.png)

So you can see everything you need to do in the same set of steps.

What do you think? Would you prefer a separate section for Build installation?

Reply:
oh yeah, it covered already, cool!

/lgtm